File: Java/UberX.java
class UberX extends Car{
    String brand;
    String model;

    public UberX(String license, Account driver, String brand, String model){//Esto es un constructor
        super(license, driver);

        this.brand= brand;
        this.model= model;
    }//Fin public UberX

    void printDataCar(){
        super.printDataCar();

        if(passegenger != null){
            System.out.println("License: " + license);
            System.out.println("name Driver: " + driver.name);
            System.out.println("Model: " + model);
            System.out.println("Brand: " + brand);
        }//Fin if
    }//Fin void

}//Fin herencia

File: PHP/uberX.php
<?php

    class UberX extends Car{
        public $brand;
        public $model;

        public function __construct($license, $driver, $brand, $model){

            parent::__construct($license, $driver);//Esto equivale al súper de Java

            $this->brand= $brand;
            $this->model= $model;
        }//Fin función constructor

    }//Fin herencia

?>